portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -82,7 +95,12 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     
     // Get the input from the form.
         String text = getParameter(request, "text-input", "");
-
+        numberOfComments = getNumberOfComments(request);
+        if (numberOfComments == -1) {
```

Comment:
Why not check if (numberOfComments < 1 || numberOfComments > 100) { // deal with error case } ?